docs/rest.md

```diff
@@ -686,6 +686,28 @@ Generally the UUID of a background operation you are interested in.
 * Operation: sync
 * Return: an object with the created username and the ssh keys imported.
 
+#### Input
+
+A javascript object with the following keys:
+* email: the email of the user to create
+* sudoers: if true adds "sudo" access to the created user
+* known: use the local system-user assertions to create the user
```

Comment:
This is a bit mysterious. Even while working on snappy I don't quite understand what would happen. Is there something we can refer to? E.g. "for more information about system-user" assertions see quux"